packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -293,7 +339,7 @@ export const ChatMessage = (React.forwardRef<HTMLDivElement, ChatMessageProps>((
   );
 
   const popperRef = React.useRef<PopperRefHandle>();
-  const { targetRef: messageRef, containerRef: actionsMenuRef } = usePopper({
```

Comment:
Can we just rename variables instead? For example:

```tsx
const { targetRef: actionsMenuTargetRef, containerRef: actionsMenuRef } = usePopper({
```

`actionsMenuRef` is very helpful to understand relationship...

Reply:
Sure, that's also fine. FWIW I'd un-destructured (restructured? :P) the variable because (to me) it helped convey that the two inner refs were related.